app/models/miq_websocket_worker/runner.rb

```diff
@@ -1,3 +1,10 @@
 class MiqWebsocketWorker::Runner < MiqWorker::Runner
   include MiqWebServerRunnerMixin
+
+  def heartbeat(*args)
+    super
+    unless worker.rails_server_options[:app].healthy?
```

Comment:
This is fine but I want to refactor this so you can ask `worker.rails_server.app.healthy`?

Maybe even have `worker.rails_server` delegate  `.healthy?` to the `app`.
It feels weird to ask options if they're healthy.